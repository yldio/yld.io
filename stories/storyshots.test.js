import initStoryshots, {
  multiSnapshotWithOptions,
  Stories2SnapsConverter
} from '@storybook/addon-storyshots'
import styleSheetSerializer from 'jest-styled-components/src/styleSheetSerializer'
import { addSerializer } from 'jest-specific-snapshot'
import renderer from 'react-test-renderer'

addSerializer(styleSheetSerializer)

initStoryshots({
  suite: 'Storyshots',
  test: multiSnapshotWithOptions({
    render: renderer
  }),
  stories2snapsConverter: new Stories2SnapsConverter({
    snapshotExtension: '.snapshot',
    storiesExtensions: ['.stories']
  })
})
