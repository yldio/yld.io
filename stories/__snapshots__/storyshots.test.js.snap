// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Form Button 1`] = `
<button
  className="sc-iwsKbI ggqLQX"
>
  Click Me
</button>
`;

exports[`Storyshots Form Checkbox 1`] = `
<section
  className="sc-gqjmRU bjIXBQ"
>
  <input
    className="sc-bZQynM eECAzN"
    id="checkbox"
    name="checkbox"
    type="checkbox"
  />
  <label
    htmlFor="checkbox"
  >
    Label
  </label>
</section>
`;

exports[`Storyshots Form Field 1`] = `
<section
  className="sc-gZMcBi eKVSBX"
>
  <input
    className="sc-gzVnrw eoVqIU"
    placeholder="test"
    type="text"
  />
</section>
`;

exports[`Storyshots Form Fieldset 1`] = `
<section
  className="sc-gqjmRU bjIXBQ"
>
  <input
    className="sc-gzVnrw eoVqIU"
    placeholder="test"
    type="text"
  />
</section>
`;

exports[`Storyshots Form Input 1`] = `
<input
  className="sc-gzVnrw eoVqIU"
  placeholder="test"
  type="text"
/>
`;

exports[`Storyshots Form Label 1`] = `
<label
  className="sc-htoDjs bJKUOL"
>
  Label
</label>
`;

exports[`Storyshots Form Textarea 1`] = `
<textarea
  className="sc-dnqmqq dKskfo"
  placeholder="test"
/>
`;

exports[`Storyshots Typography Large Body 1`] = `
<h3
  className="sc-bwzfXH cSnxzX"
>
  Large Body
</h3>
`;

exports[`Storyshots Typography Large title 1`] = `
<h1
  className="sc-bdVaJa eLsBGr"
>
  Large title
</h1>
`;

exports[`Storyshots Typography Small Title 1 1`] = `
<h4
  className="sc-htpNat gmqdcE"
>
  Small Title
</h4>
`;

exports[`Storyshots Typography Small Title 2 1`] = `
<h5
  className="sc-bxivhb bzAYeX"
>
  Small Title
</h5>
`;

exports[`Storyshots Typography Small Title 3 1`] = `
<h6
  className="sc-ifAKCX iMlpgY"
>
  Small Title
</h6>
`;
